pom.xml

```diff
@@ -64,6 +64,12 @@
             <version>${junit.platform.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Please document new OSS dependency in the README for attribution.

Reply:
✅